OffchainLabs/arbitrum
# Arbos testing

- Add create2 test
- Add convenient deposit method to ethbridge buidler config
- Update ArbOS with https://github.com/OffchainLabs/arb-os/pull/203
- Update snapshot system to support a limited set of recent snapshots
- Change validator docker file to use Debian instead of alpine to fix crash and simplify build. This brings our validator docker build in line with our circleci docker build which should make maintenance easier
